### PR TITLE
[MRG+1] Add sphinx_rtd_theme to docs setup readme

### PR DESCRIPTION
**Pull request by harshasrinivas**
Concerns Sphinx 1.4 release #1893 . `sphinx_rtd_theme` has to be installed manually.

### CODE REVIEW

**Comment by redapple** [comment]
What about adding a `requirements.txt` to `scrapy/docs/` and referencing if in `tox.ini's [docs]`?

**Comment by harshasrinivas** [comment]
@redapple Yes, I too thought of the same. :) 
But since there are just 2 requirements for building the docs, do you think adding a requirements.txt would be beneficial?

**Comment by redapple** [comment]
I think it would still be easier to `pip install -r docs/requirements.txt`

**Comment by harshasrinivas** [comment]
Sure @redapple , adding right away.

File: docs/README.rst [hunk 1]
```diff
@@ -15,7 +15,7 @@ and all its dependencies run
 
 ::
 
-    pip install 'Sphinx >= 1.3' sphinx_rtd_theme
+    pip install -r requirements.txt
```
Comment (redapple) [inline docs/README.rst]: I'm not sure that it's clear that you would run this inside `scrapy/docs/`

Comment (harshasrinivas) [inline docs/README.rst]: Just like the `Compile the documentation` part of this README, to maintain uniformity shall I mention "run the following command from this dir"?

Comment (redapple) [inline docs/README.rst]: That would not hurt indeed.

**Pull request merged**